src/drt/src/io/GuideProcessor.cpp

```diff
@@ -34,7 +34,7 @@
 
 #include "frProfileTask.h"
 namespace drt::io {
-
+using Interval = boost::icl::interval<frCoord>;
```

Comment:
Why not use this in the definition of TrackIntervals?

Reply:
TrackIntervals holds interval_set not interval

Reply:
Yes but you defined it as
```
using TrackIntervals = std::map<frCoord, boost::icl::interval_set<frCoord>>;
```
while it could be 
```
using Interval = boost::icl::interval<frCoord>;
using TrackIntervals = std::map<frCoord, Interval>;
```
to make the relationship clear.

Reply:
my bad